test/e2e/major_upgrade_v2.go

```diff
@@ -12,28 +12,40 @@ import (
 	v1 "github.com/celestiaorg/celestia-app/v3/pkg/appconsts/v1"
 	v2 "github.com/celestiaorg/celestia-app/v3/pkg/appconsts/v2"
 	"github.com/celestiaorg/celestia-app/v3/test/e2e/testnet"
+	"github.com/celestiaorg/knuu/pkg/knuu"
 	"github.com/tendermint/tendermint/rpc/client/http"
 )
 
 func MajorUpgradeToV2(logger *log.Logger) error {
 	var (
+		testName      = "MajorUpgradeToV2"
 		numNodes      = 4
 		upgradeHeight = int64(10)
 	)
```

Comment:
[nit] this codebase doesn't make use of `var` blocks inside of function bodies so I would've expected to see

```suggestion
	testName := "MajorUpgradeToV2"
	numNodes := 4
	upgradeHeight := int64(10)
```

which is conveniently 2 fewer lines 👍

Reply:
:) it is kinda subjective. I think the first one looks cleaner, but don't have a strong opinion on it.